staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -70,6 +74,16 @@ const (
 	dbMetricsMonitorJitter = 0.5
 )
 
+var (
+	// This is a warning error that we defined and will print the message for given versions
+	// of the cluster.
+	ErrDeprecatedCluster = errors.New("warning: Your cluster is using a deprecated etcd version (<3.4.25 or v3.5.[0-7]). To prevent compatibility issues, the 'ConsistentListFromCache' feature has been disabled (see github.com/etcd-io/etcd#15220)")
```

Comment:
Issue github.com/etcd-io/etcd#15220 doesn't have full context on why consistent reads from cache would not work, maybe linking to https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md#bug-in-etcd-progress-notification
 is a better idea.

Reply:
I think the link is a bit too long, i tried to make it shorter using kep.k8s.io but i couldn't redirect to the page, it goes to issue. But i put it there if there is any thing wrong it please let me know.